packages/viewer/src/components/viewer-button/viewer-button.tsx

```diff
@@ -0,0 +1,18 @@
+import { Component, h, Host } from '@stencil/core';
+
+@Component({
+  tag: 'vertex-viewer-button',
+  styleUrl: 'viewer-button.css',
+  shadow: true,
+})
+export class ViewerButton {
```

Comment:
I'm leaning towards not including this component for now.

Reply:
I do see the usefulness though. hard to imagine a scenario where a toolbar button should have the default button styles.

Reply:
Ok, will keep it around. I think we'll need it in the future anyway for buttons that have richer logic, like selected buttons.